src/panels/lovelace/views/hui-view.ts

```diff
@@ -54,7 +54,11 @@ export class HUIView extends ReactiveElement {
   // Public to make demo happy
   public createCardElement(cardConfig: LovelaceCardConfig) {
     const element = createCardElement(cardConfig) as LovelaceCard;
-    element.hass = this.hass;
+    try {
+      element.hass = this.hass;
+    } catch {
```

Comment:
Not all browsers support optional catch binding; (Safari)

Reply:
Does that mean `catch {` rather than `catch (error) {`?

Guess I need to mark the error as ignored by the linter somehow then?

Reply:
Yes, that is what it means :-)

You can do:

`} catch (_e: any) {`